lib/tbot/identity/identity.go

```diff
@@ -435,6 +435,27 @@ func LoadIdentity(d bot.Destination, kinds ...ArtifactKind) (*Identity, error) {
 			return nil, trace.WrapWithMessage(err, "could not read artifact %q from destination %s", artifact.Key, d)
 		}
 
+		// Attempt to load from an old key if there was no data in the current
+		// key. This will be in the case as d.Read for the file destination will
+		// not throw an error if the file does not exist.
+		// This allows migrations of key names.
+		if artifact.OldKey != "" && len(data) == 0 {
+			log.Debugf(
+				"Unable to load from current key %q, trying to migrate from old key %q",
+				artifact.Key,
+				artifact.OldKey,
+			)
+			data, err = d.Read(artifact.OldKey)
+			if err != nil {
+				return nil, trace.WrapWithMessage(
```

Comment:
```suggestion
				return nil, trace.Wrap(
```

tiny nit, I'm told `trace.WrapWithMessage(...)` is deprecated in favor of just `trace.Wrap()`, which also accepts formatting params.